tests/migration/deregistration_system.pm

```diff
@@ -0,0 +1,18 @@
+# Copyright 2023 SUSE LLC
```

Comment:
file should be called "deregister_system.pm" to follow "verb+noun" convention used for other files.